exonum/src/messages/macros.rs

```diff
@@ -24,16 +24,16 @@
 macro_rules! impl_serde_hex_for_binary_value {
     ($name:ident) => {
         impl hex::ToHex for $name {
-            fn write_hex<W: std::fmt::Write>(&self, w: &mut W) -> std::fmt::Result {
+            fn encode_hex<T: ::std::iter::FromIterator<char>>(&self) -> T {
```

Comment:
It's not necessary to write `::` in Rust 2018

Reply:
Strange that clippy haven't noticed about that. But good catch anyway 👍🏻